tests_integration/__tests__/__snapshots__/config-invalid.js.snap

```diff
@@ -12,4 +12,9 @@ exports[`throw error with invalid config option (trailingComma) 1`] = `
 "
 `;
 
+exports[`throw error with invalid config precedence option (string) 1`] = `
+"Invalid configuration:
```

Comment:
This error is incorrect.

Reply:
What would be the expected snapshot ? What part of the error is incorrect ? the `(string)` ?

Reply:
Let me post the steps I think is required to solve this and you can correct if i am wrong

1. create a file : `tests_integration/cli/config/invalid/option//precedence`
2. updtate the assertion message to "throw error with invalid config precedence option (precedence)"
3. generate new snapshots and clear the old one

Is the steps above correct ?

Reply:
and on `tests_integration/cli/config/invalid/option//precedence` having this contents:

```JSON
{
    "config-precedence": "invalidValue"
}
```

Reply:
Sorry for not being clearer, all I meant was the error produced by Prettier by passing `--config-precedence=blarg` should be `Invalid value for --config-precedence`, not `Invalid configuration`.